core/services/fluxmonitor/flux_monitor.go

```diff
@@ -736,17 +740,19 @@ func (p *PollingDeviationChecker) processLogs() {
 			continue
 		}
 
+		ctx, cancel = postgres.DefaultQueryCtx()
+		defer cancel()
 		switch log := broadcast.DecodedLog().(type) {
 		case *flux_aggregator_wrapper.FluxAggregatorNewRound:
 			p.respondToNewRoundLog(*log)
-			err = broadcast.MarkConsumed()
+			err = p.logBroadcaster.MarkConsumed(p.store.DB.WithContext(ctx), broadcast)
 			if err != nil {
 				logger.Errorf("Error marking log as consumed: %v", err)
 			}
 
 		case *flux_aggregator_wrapper.FluxAggregatorAnswerUpdated:
 			p.respondToAnswerUpdatedLog(*log)
-			err = broadcast.MarkConsumed()
+			err = p.logBroadcaster.MarkConsumed(p.store.DB.WithContext(ctx), broadcast)
```

Comment:
Does `p.respondToAnswerUpdatedLog()` sometimes insert an eth_tx? Would we want that to be transactional?

Reply:
I didn't bother making the v1 FM stuff transactional also, itd be a fairly large all throwaway restructuring. Only did it for v2